src/layout/strategy/withinContainer.js

```diff
@@ -0,0 +1,41 @@
+import d3 from 'd3';
+import {rebindAll} from '../../util/rebind';
+
+export default function() {
+
+    var containerWidth = 1,
```

Comment:
With a couple of couple of different types of rectangle in action (container and children) I  think I'd prefer to be explicit.